### DISCUSSION TITLE
Error when running Github Copilot on VSCode

### DISCUSSION THREAD

**Discussion by Farissoliman** [community/community#53039] [Copilot Conversations]
### Select Topic Area

Bug

### Body

After authenticating on VSCode, I try running github copilot and using the test functions on the initialization page and I keep getting this error. I tried downgrading, deleting copilot, signing out from github on VSCode, etc. but nothing seemed to work. Any ideas on how to fix it?

The error message in question:
`Command interactiveEditor.start appears multiple times in the commands section.`

**Comment by ghost** [comment]
The same error is noted here: https://github.com/orgs/community/discussions/52604
and here: https://github.com/orgs/community/discussions/51923

Both were solved by downgrading to Version `1.78.x` if you've yet to try that.

**Reply by Farissoliman** [reply]
Downgrading didn't work for me. Still nothing shows in the output tab on VSCode and the only message I get in the runtime status tab on the CoPilot extension window after downgrading is this:
`Activation Event:onStartupFinished`
`Activation Time:80ms`

**Reply by ghost** [reply]
> Downgrading didn't work for me. Still nothing shows in the output tab on VSCode and the only message I get in the runtime status tab on the CoPilot extension window after downgrading is this: `Activation Event:onStartupFinished` `Activation Time:80ms`

Are you perhaps on a Windows system? If so [this message](https://github.com/orgs/community/discussions/52387#discussioncomment-5634933) from staff [Akash1134](https://github.com/Akash1134) may be relevant. It might also help to share more context (such as the entire error output) if possible.

**Reply by Farissoliman** [reply]
I forgot to mention, but I am running this on Mac OS. My VSCode is also completely up to date. There is no error message showing up when I open the output tab and I'm not getting any error messages other than the one mentioned above (`Command interactiveEditor.start appears multiple times in the commands section.`)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Farissoliman ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
